### ISSUE TITLE
Permissions and cache on struct data

### ISSUE THREAD

**Issue by mtkirchner** [cosmocode/dokuwiki-plugin-struct#496]
I'm trying to find a reason for the eratic amount of information displayed by an aggregation.

My application is a first-aid log, that can be filled by every user, but entries should only be visible to management. I use a form to add data to a schema, using runas and different namespaces to ceate the data in, depending on the part of the company. Each of the namespaces then has different permissions, which, to my understanding, would reflect on the data created on pages within these namespaces.

Basically it works. But when I switch between different logins to verify the permissions, strange things happen. The last days I tried to find the problem within the permissions, but only today I thought about caching and adding a ~~NOCACHE~~ seems to stop my problem.

Could it be this simple? If so, I would file caching with structured data as a bug, since I would never have expected that to happen.
